src/cli.yaml

```diff
@@ -69,3 +69,16 @@ subcommands:
       about: Stops Valheim
       version: "1.1"
       author: mbround18
+  - backup:
+      about: Backups the current saves to a specifc location
+      version: "1.0"
+      author: mbround18
+      args:
+        - INPUT:
+            about: Directory to backup
+            required: true
+            index: 1
+        - OUTPUT:
+            about: Sets the input file to use
```

Comment:
Nit, but should this be something along the lines of filename or filepath for the gzipped backup?

Reply:
Yeah, I can update the about section